src/scss/vendors/_vendors.scss

```diff
@@ -31,6 +31,59 @@
 	}
 }
 
+.fl-input-enhanced #defaultContainer .fl-prefsEditor-separatedPanel .fl-panelBar .fl-prefsEditor-buttons {
+	display: inline-block;
+	height: rem(80);
+	padding-top: 0;
+
+	button {
+		padding: rem(10) 0;
+	}
+
+	.fl-prefsEditor-reset {
+		border: none;
+		margin-left: rem(8);
+		padding-bottom: rem(5);
+	}
+
+	button[aria-expanded="false"] {
+		margin-top: rem(20);
+	}
+
+	button[aria-expanded="true"] {
+		border-bottom: 2px solid #ccc;
+		margin-left: rem(8);
+		padding-left: rem(10);
+	}
+}
+
+@media screen and (min-width: 40em) {
```

Comment:
Add a code comment to explain this media query breakpoint comes from the infusion preferences framework for UIO in order to customize UIO button styles on desktop and mobile.